addons/account/__init__.py

```diff
@@ -5,6 +5,9 @@
                   'CD', 'SN', 'TD', 'TG']
 VAT_LIST = ['AT', 'BE', 'CA', 'CO', 'DE', 'EC', 'ES', 'ET', 'FR', 'GR', 'IT', 'LU', 'MX', 'NL',
             'NO', 'PL', 'PT', 'RO', 'SI', 'TR', 'GB', 'VE', 'VN']
+PEPPOL_LIST = ['AD', 'AL', 'AT', 'BA', 'BE', 'BG', 'CH', 'CY', 'CZ', 'DE', 'DK', 'EE', 'ES', 'FI',
```

Comment:
I'm not sure I like that it's in this file, but I couldn't find a better place for it...

Reply:
Well, this was the only file that already contained such country lists, so ... I couldn't find a better place either 😅